xmltotxt.py

```diff
@@ -22,10 +23,24 @@ def main():
         os.makedirs(out_dir)
 
     if not os.access(out_dir, os.W_OK):
-        print("%s folder is not writeable.")
+        print("%s folder is not writeable." % out_dir)
         sys.exit()
+    
+    class_file = os.path.join(os.path.dirname(os.path.realpath('__file__')), args.c)
 
-    transformer = Transformer(xml_dir=xml_dir, out_dir=out_dir)
+    if not os.access(class_file, os.F_OK):
+        print("%s file is missing." % class_file)
+        sys.exit()
+
+    if not os.access(class_file, os.R_OK):
+        print("%s file is not readable." % class_file)
+        sys.exit()
+    
+    if not os.path.splitext(class_file)[-1] == ".txt":
```

Comment:
We should be able to read files without extensions. Please, get rid of this part of the code.

Reply:
Sure!